src/plugins/kibana_utils/common/of.ts

```diff
@@ -15,6 +15,7 @@
  *
  * @param promise Promise to convert to 3-tuple.
```

Comment:
I think this was supposed to be "thrupple" ;)

Reply:
Tripel :D 

![image](https://user-images.githubusercontent.com/7784120/146574202-c863b562-7911-4a31-9f50-73250cb28b8a.png)